human_analysis/1_process_recount2_data.ipynb

```diff
@@ -4718,21 +3059,15 @@
    "source": [
     "# Setup directories\n",
     "# Create VAE directories\n",
-    "output_dirs = [os.path.join(base_dir, dataset_name, \"models\"),\n",
-    "               os.path.join(base_dir, dataset_name, \"logs\")]\n",
-    "\n",
-    "# Check if analysis output directory exist otherwise create\n",
-    "for each_dir in output_dirs:\n",
-    "    if os.path.exists(each_dir) == False:\n",
-    "        print('creating new directory: {}'.format(each_dir))\n",
-    "        os.makedirs(each_dir, exist_ok=True)\n",
```

Comment:
This section is not necessary, because `os.makedirs(dirname, exist_ok=True)` will always generate the parent directories if they don't exist yet.